test/module/shared_model/builders/protobuf/test_transaction_builder.hpp

```diff
@@ -21,21 +21,19 @@
 #include "builders/protobuf/builder_templates/transaction_template.hpp"
 #include "module/shared_model/validators/validators.hpp"
 
-using namespace shared_model::proto;
-using namespace shared_model::validation;
-
 /**
  * Builder alias, to build shared model proto transaction object avoiding
  * validation and "required fields" check
  */
-using TestTransactionBuilder =
-    TemplateTransactionBuilder<(1 << TemplateTransactionBuilder<>::total) - 1,
-                               TransactionAlwaysValidValidator,
-                               Transaction>;
+using TestTransactionBuilder = shared_model::proto::TemplateTransactionBuilder<
```

Comment:
don't make unrelated refactorings plz

Reply:
Why not?

Reply:
the Lesser merge conflicts the happier developers are